unit-tests/func/func-common.h

```diff
@@ -94,3 +135,30 @@ inline stream_profile find_confidence_corresponding_to_depth( rs2::depth_sensor
     REQUIRE( confidence_profile != stream_profiles.end() );
     return *confidence_profile;
 }
+
+
+bool get_profile_by_stream_parameters( rs2::depth_sensor ds,
+                                       rs2_stream stream,
+                                       rs2_format format,
+                                       int width,
+                                       int height,
+                                       rs2::stream_profile & required_profile )
+{
+    auto profiles = ds.get_stream_profiles();
+    auto vga_depth_profile
+        = std::find_if(profiles.begin(), profiles.end(), [=](rs2::stream_profile sp) {
+        auto vp = sp.as< rs2::video_stream_profile >();
+        return ((sp.stream_type() == stream) && (sp.format() == format)
```

Comment:
What about 'any'? Do we want to support it?

Reply:
Good idea, I added ANY support for rs2_stream / rs2_format and change depth sensor parameter to sensor,
I didn't add "ANY" support for width + height for now, if someone will need it in the future he can implement it with 0 / -1 input.